From 21cdd9d73e2bfa79a1df93884b617fded0619d8f Mon Sep 17 00:00:00 2001
Message-ID: <21cdd9d73e2bfa79a1df93884b617fded0619d8f.1708289807.git.sharlatanus@gmail.com>
In-Reply-To: <b6807bf457aa3debcd60c49908ebd1b147744e20.1708289807.git.sharlatanus@gmail.com>
References: <b6807bf457aa3debcd60c49908ebd1b147744e20.1708289807.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 17 Feb 2024 00:35:59 +0000
Subject: [PATCH v2 07/10] gnu: go-github-com-songmu-gitconfig: Move to
 golang-xyz.

* gnu/packages/golang.scm (go-github-com-songmu-gitconfig): Move from
here ...
* gnu/packages/golang-xyz.scm: ... to here.

* gnu/packages/version-control.scm: Add (gnu packages golang-xyx)
module.

Change-Id: Ibab2595d981d46096501bb8ab495e899d2aacf7d
---
 gnu/packages/golang-xyz.scm      | 28 ++++++++++++++++++++++++++++
 gnu/packages/golang.scm          | 26 --------------------------
 gnu/packages/version-control.scm |  1 +
 3 files changed, 29 insertions(+), 26 deletions(-)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index c64200d365..a6031a8354 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -3,6 +3,7 @@
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2019 Brian Leung <bkleung89@gmail.com>
 ;;; Copyright © 2020 Efraim Flashner <efraim@flashner.co.il>
+;;; Copyright © 2020 Joseph LaFreniere <joseph@lafreniere.xyz>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
 ;;; Copyright © 2021 Raghav Gururajan <rg@raghavgururajan.name>
 ;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
@@ -1019,6 +1020,33 @@ (define-public go-github-com-songgao-water
 Use waterutil with it to work with TUN/TAP packets/frames.")
     (license license:bsd-3)))
 
+(define-public go-github-com-songmu-gitconfig
+  (package
+    (name "go-github-com-songmu-gitconfig")
+    (version "0.1.0")
+    (home-page "https://github.com/songmu/gitconfig")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url home-page)
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1y01h496a7pfj1g2bclls5b0nl3vnj7nz610jj1dzq9kxrwxk7fk"))))
+    (build-system go-build-system)
+    (arguments
+     `(#:import-path "github.com/Songmu/gitconfig"
+       ;; Package's tests appear to be hardcoded to the author's gitconfig
+       ;; and require network access.
+       #:tests? #f))
+    (propagated-inputs
+     (list go-github-com-goccy-yaml))
+    (synopsis "Go library to get configuration values from gitconfig")
+    (description
+     "@{gitconfig} is a package to get configuration values from gitconfig.")
+    (license license:expat)))
+
 (define-public go-github-com-stathat-go
   (let ((commit "74669b9f388d9d788c97399a0824adbfee78400e")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 59d0df0781..51f1c4d954 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -8953,32 +8953,6 @@ (define-public go-github-com-tekwizely-go-parsing
 parsers, and related tools.")
       (license license:expat))))
 
-(define-public go-github-com-songmu-gitconfig
-  (package
-    (name "go-github-com-songmu-gitconfig")
-    (version "0.1.0")
-    (home-page "https://github.com/songmu/gitconfig")
-    (source
-     (origin
-       (method git-fetch)
-       (uri (git-reference
-             (url home-page)
-             (commit (string-append "v" version))))
-       (file-name (git-file-name name version))
-       (sha256
-        (base32 "1y01h496a7pfj1g2bclls5b0nl3vnj7nz610jj1dzq9kxrwxk7fk"))))
-    (build-system go-build-system)
-    (arguments
-     `(#:import-path "github.com/Songmu/gitconfig"
-       ;; Package's tests appear to be hardcoded to the author's gitconfig
-       ;; and require network access.
-       #:tests? #f))
-    (propagated-inputs
-     (list go-github-com-goccy-yaml))
-    (synopsis "Go library to get configuration values from gitconfig")
-    (description "@{gitconfig} is a package to get configuration values from gitconfig.")
-    (license license:expat)))
-
 (define-public go-github-com-akosmarton-papipes
   (let ((commit "3c63b4919c769c9c2b2d07e69a98abb0eb47fe64")
         (revision "0"))
diff --git a/gnu/packages/version-control.scm b/gnu/packages/version-control.scm
index afef900bbb..a7ac89b7d9 100644
--- a/gnu/packages/version-control.scm
+++ b/gnu/packages/version-control.scm
@@ -110,6 +110,7 @@ (define-module (gnu packages version-control)
   #:use-module (gnu packages golang-check)
   #:use-module (gnu packages golang-crypto)
   #:use-module (gnu packages golang-web)
+  #:use-module (gnu packages golang-xyz)
   #:use-module (gnu packages groff)
   #:use-module (gnu packages guile)
   #:use-module (gnu packages guile-xyz)
-- 
2.41.0

